packages/rich-text-from-markdown/package.json

```diff
@@ -53,7 +53,7 @@
     "rollup-plugin-sourcemaps": "^0.6.3",
     "rollup-plugin-typescript2": "^0.36.0",
     "ts-jest": "^29.1.2",
-    "tslib": "^2.3.1",
+    "tslib": "^2.6.2",
     "tslint": "^6.1.3",
     "typescript": "^4.4.2"
```

Comment:
Why not updating these packages as well?

Reply:
Upps, on it!